src/Solvers/batched_tridiagonal_solver.jl

```diff
@@ -2,6 +2,8 @@ using Oceananigans.Architectures: device_event, arch_array
 
 import Oceananigans.Architectures: architecture
 
+import Oceananigans.Utils: launch!
+
```

Comment:
?

Reply:
launch! is imported from AMDGPU as well. I think @christophernhill was looking at resolving the import issue.

Reply:
One way to resolve this is to avoid writing `using AMDGPU`. Instead just import the spcific names you need:

```julia
using AMDGPU: ROCmArray
```

etc.

I think it's also nicer because it's more explicit about where those names come from.